build/deps/gen/dep_clang_tidy_darwin_arm64.MODULE.bazel

```diff
@@ -0,0 +1,10 @@
+# WARNING: THIS FILE IS AUTOGENERATED BY update-deps.py DO NOT EDIT
+
+http_file = use_repo_rule("@//:build/http.bzl", "http_file")
```

Comment:
nit: `http_file` (and perhaps other rules) are duplicated many times in these definition files, I'd rather have them be defined once in MODULE.bazel so that we don't need them in each generated file. (I'm assuming here that files pulled in with include() can use definitions from the main module file).

Reply:
I'm pretty sure the symbol is not visible across includes but will recheck

Reply:
Yes, confirmed it does not work.
```
ERROR: /Users/npaun/cod/workerd/build/deps/gen/dep_buildifier_darwin_arm64.MODULE.bazel:3:1: name 'http_file' is not defined
ERROR: Error computing the main repository mapping: syntax error in MODULE.bazel file for <root>
```